sample-app/android/app/fabric.properties

```diff
@@ -0,0 +1,3 @@
+# Fabric properties file: app/fabric.properties
```

Comment:
This files should be in `.gitignore`! remove it please

Reply:
👍